integration-tests/config.ts

```diff
@@ -75,120 +76,86 @@ const defaultSecretKey: SecretKeyConfig = {
+    rpcCacheMilliseconds: process.env[`RPC_CACHE_MILLISECONDS`] || '1000',
+    knownBaker: process.env[`TEZOS_BAKER`] || 'tz1cjyja1TU6fiyiFav3mFAdnDsCReJ12hPD',
+    knownContract: process.env[`TEZOS_${networkName}_CONTRACT_ADDRESS`] || knownContracts.contract,
+    knownBigMapContract: process.env[`TEZOS_${networkName}_BIGMAPCONTRACT_ADDRESS`] || knownContracts.bigMapContract,
+    knownTzip1216Contract: process.env[`TEZOS_${networkName}_TZIP1216CONTRACT_ADDRESS`] || knownContracts.tzip12BigMapOffChainContract,
+    knownSaplingContract: process.env[`TEZOS_${networkName}_SAPLINGCONTRACT_ADDRESS`] || knownContracts.saplingContract,
+    txRollupWithdrawContract: process.env[`TEZOS_${networkName}_TX_ROLLUP_WITHDRAW_CONTRACT`] || '',
+    txRollupDepositContract: process.env[`TEZOS_${networkName}_TX_ROLLUP_DEPOSIT_CONTRACT`] || '',
+    knownViewContract: process.env[`TEZOS_${networkName}_ON_CHAIN_VIEW_CONTRACT`] || knownContracts.onChainViewContractAddress,
+    txRollupAddress: process.env[`TEZOS_${networkName}_TXROLLUP_ADDRESS`] || knownContracts.txRollupAddress,
+    protocol: protocol,
+    signerConfig: signerConfig,
+  }
+}
 
-const limanetSecretKey = {
-  rpc: process.env['TEZOS_RPC_LIMANET'] || 'http://ecad-limanet-full:8732',
-  pollingIntervalMilliseconds: process.env['POLLING_INTERVAL_MILLISECONDS'] || undefined,
-  rpcCacheMilliseconds: process.env['RPC_CACHE_MILLISECONDS'] || '1000',
-  knownBaker: process.env['TEZOS_BAKER'] || 'tz1cjyja1TU6fiyiFav3mFAdnDsCReJ12hPD',
-  knownContract: process.env['TEZOS_LIMANET_CONTRACT_ADDRESS'] || knownContractPtLimaPtL,
-  knownBigMapContract: process.env['TEZOS_LIMANET_BIGMAPCONTRACT_ADDRESS'] || knownBigMapContractPtLimaPtL,
-  knownTzip1216Contract: process.env['TEZOS_LIMANET_TZIP1216CONTRACT_ADDRESS'] || knownTzip12BigMapOffChainContractPtLimaPtL,
-  knownSaplingContract: process.env['TEZOS_LIMANET_SAPLINGCONTRACT_ADDRESS'] || knownSaplingContractPtLimaPtL,
-  txRollupWithdrawContract: process.env['TEZOS_LIMANET_TX_ROLLUP_WITHDRAW_CONTRACT'] || '',
-  txRollupDepositContract: process.env['TEZOS_LIMANET_TX_ROLLUP_DEPOSIT_CONTRACT'] || '',
-  knownViewContract: process.env['TEZOS_LIMANET_ON_CHAIN_VIEW_CONTRACT'] || knownOnChainViewContractAddressPtLimaPtL,
-  txRollupAddress: process.env['TEZOS_LIMANET_TXROLLUP_ADDRESS'] || txRollupAddressPtLimaPtL,
-  protocol: Protocols.PtLimaPtL,
-  signerConfig: defaultSecretKey
-};
+const kathmandunetEphemeral: Config =
+  defaultConfig({
+    networkName: 'KATHMANDUNET',
+    protocol: Protocols.PtKathman,
+    defaultRpc: 'http://ecad-kathmandunet-archive.i.tez.ie:8732',
+    knownContracts: knownContractsPtKathman,
+    signerConfig: defaultEphemeralConfig('https://api.tez.ie/keys/kathmandunet')
+  });
+
+const kathmandunetSecretKey: Config =
+  { ...kathmandunetEphemeral, ...{ signerConfig: defaultSecretKey } };
+
+const limanetEphemeral: Config =
+  defaultConfig({
+    networkName: 'LIMANET',
+    protocol: Protocols.PtJakart2,
```

Comment:
Wrong protocol hash -> should be lima, not jakarta

Reply:
Fixed in the latest version.